setup.py

```diff
@@ -217,7 +217,7 @@ def run(self):
              'vetoes.chisq']
 ext = []
 cython_compile_args = ['-O3', '-w', '-ffast-math',
-                       '-ffinite-math-only']
+                       '-fno-finite-math-only']
```

Comment:
Okay, the CI is now passing (yay), but I am concerned about having had to make this change. I don't understand why it hasn't also affected the main branch. The underlying issue is a change in a core library (I think libm) to remove the explicit finite math functions (I'm not clear if this is now deprecated and this flag does nothing, or if they are implemented differently). However, it seems that the code is built against one version (of libm), but then links against a different version when running, which then causes a missing flag failure.

I don't think this will change much. It *may* slightly impact performance of the cython code, but I suspect it would be marginal if noticeable at all.